### PR TITLE
DOC-1729-mlwb1.3-release-notes 1st draft

### PR DESCRIPTION
**Pull request by victorleeTG**
1st draft. Questions:
1. Are the links correct/reasonable?
2. Is the distinction between Data Loader and Data Loader Factory correct?
3. Can you suggest links for the remaining new/updated features?

As a separate issue, we also need brief Release Notes for v1.2

### CODE REVIEW

File: modules/faq/pages/release-notes.adoc [hunk 1]
```diff
@@ -1,5 +1,29 @@
 = Release Notes
 
+== TigerGraph ML Workbench 1.3
+
+=== New features
+
+* Two new xref:1.3@pytigergraph:gds:dataloaders.adoc[data loader classes]: `HGTLoader` and `NodePiece`.
+* Callback functions to all xref:1.3@pytigergraph:gds:gds.adoc[data loader factories]: users can write functions to process the batch in a background thread before it is passed into the training loop.
+* A delimiter parameter to all xref:1.3@pytigergraph:gds.adoc[data loader factories]: users can choose what delimiter they want to separate attributes they are loading from the graph.
+* `customizeHeader()` function to change default timeout and response size values.
```
Comment (billshitg) [inline modules/faq/pages/release-notes.adoc]: @parkererickson , is this customizeHeader function included in the docs?

**Review by billshitg — APPROVED** [review]

**Comment by billshitg** [comment]
All look good to me, with one place where we might want to add a link (commented inline).

**Comment by billshitg** [comment]
Regarding the release notes for v1, looks like we need it for 1.2, as 1.1 and 1.0 are there. Would you like me to create a separate DOC ticket for it? Thanks!

File: modules/faq/pages/release-notes.adoc (hunk 1 above)
Comment (parkererickson-tg) [reply modules/faq/pages/release-notes.adoc]: Oh, no. `customizeHeader()` was part of a patch version I believe. Let me double check though, we can release it as a v1.4 feature too.

Comment (billshitg) [reply modules/faq/pages/release-notes.adoc]: Sounds good. Let's remove this line then. Thanks!

Comment (victorleeTG) [reply modules/faq/pages/release-notes.adoc]:
```suggestion
```

Comment (victorleeTG) [reply modules/faq/pages/release-notes.adoc]: Will delete customizeHeader() from 1.3 release notes

**Pull request merged**